docs/API/InstrumentFormat.md

```diff
@@ -18,6 +23,37 @@ this standard without breaking backwards compatibility. Instruments MAY implemen
 the instrument format described below and the accompagnying rules format in the same
 JSON object, or they MAY be used independently.
 
+Language specifications can be added for fields that display text (LongName, Description, Values, Labels) in the following format:
+
+```js
+...
+    "RegularText" : {
+        "lang-1" : "text in lang 1",
+        "lang-2" : "text in lang 2",
+        ...
+    },
+    "Values" : {
+        "lang-1" : {
+            "val1" : "0"
+            "val2" : "1"
+        },
+        "lang-2" : {
+            "val1" : "10"
+            "val2" : "9"
+        },
+        ...
+    },
+...
+```
+
+The text displayed will depend on the candidate's preferred language parameter. The default is `"en-ca"`.
```

Comment:
It's fairly standard to capitalize the variant of the language (though I'm not sure what the name of the RFC or IETF standard is for language codes...). I think it should be `en-CA`

Reply:
language in lowercase and country code in uppercase